.github/workflows/test.yml

```diff
@@ -37,16 +42,16 @@ jobs:
       with:
         filters: |
           src-esp32:
-            - 'src/esp32/**'
+            - 'src/esp3233/**'
```

Comment:
Was this maybe left over from some debugging?

Reply:
yes I'll remove them before merge